logstash-core/lib/logstash/pipeline.rb

```diff
@@ -323,61 +393,70 @@ def flush_filters(options = {}, &block)
     end
   end
 
+  class FlusherObserver
+    def initialize(logger, flushing)
+      @logger = logger
+      @flushing = flushing
+    end
+
+    def update(time, result, exception)
```

Comment:
it's not clear what the `FlusherObserver` class is meant for nor what the `update` method parameter relates to... maybe  add documentation?